### ISSUE TITLE
Save population with VSPs and continue to evolve

### ISSUE THREAD

**Issue by khornick1** [BoPeng/simuPOP#82]
Hi,

I would like to run a number of scenarios (with age structure and overlapping gens) starting from the same burnin. My plan was to run the burnin and use the `save.pop` function, and then to load and continue to evolve that population depending on the scenario. While I am able to do this with vsps for each age class, I am unable to do this with more coarse vsps to define age classes. When I load the population it recognizes that there are ages, but doesn't use that info to split vsps:

`pop = sim.loadPopulation('burnin_gen_19.pop')
print(pop.infoFields())`

`('age', 'ind_id', 'father_id', 'mother_id', 'gen_id', 'migrate_to', 'migrate_from')
`

`pop.setVirtualSplitter(sim.InfoSplitter(field='age', cutoff=[1,2,12]))
`
When I plot a histogram of the subpopulation ages it shows one age class only, whereas before when I plotted the histogram the number in each age class/vsp was recognized.

I see that you specify that they can't be saved here [http://simupop.sourceforge.net/manual_release/build/userGuide_ch4_sec3.html#save-and-load-a-population](url), but I'm wondering if there is a way to do simulations with a burnin (including age structure and overlapping gens), save that burnin, and run diff scenarios from that burnin over time using different python scripts? 

Thanks in advance for your help.

Katie

**Comment by BoPeng** [comment]
I do not quite understand the question. Yes, it is by design that VSPs are not saved with population, because VSPs are ways to "view" the population, not the population itself. However, if you `loadPopulation`, and then `setVirtualSplitter`, the population should have the same VSPs as before they are saved. To verify this, you can simply dump the age and see its distribution with `pop.indInfo('age')`, and see if there should be individuals in each VSP.

**Comment by khornick1** [comment]
I apologize for the confusion. The question is: can I run a burnin, save pop from burnin, then load the saved pop from the burnin and continue to evolve it?––and you answered just that. I've done it before, but was having some errors this time around even when I dumped and tried to hist the distribution. I realized it may have been because of something silly––after I loaded the population I used `pop.setInfoFields(['age', 'ind_id', 'father_id', 'mother_id', 'gen_id', 'migrate_to', 'migrate_from']) 
`
Now it seems to work without that line of code. I just wanted to make sure it was something I am able to do, since I've been running a burnin for several days now and all of a sudden I'm getting errors. Sorry for the confusion and thanks for your continued help!

Take care,
Katie
